docker-compose.yml

```
@@ -0,0 +1,36 @@
+version: '3.8'
+
+name: untapped_scrapper
+
+services:
+  untapped:
+    container_name: scrapper-container
+    image: scrapper-image
+    build:
+      context: .
+    env_file:
+      - .env
+    volumes:
+      - ./:/src
+    depends_on:
+      mysql-db:
+        condition: service_healthy
+
+  mysql-db:
+    container_name: database-container
+    image: mysql:8.0
+    cap_add:
+      - SYS_NICE
+    env_file:
+      - .env
+    ports:
+      - ${MYSQL_PORT}:${MYSQL_PORT}
```

```suggestion
      - ${MYSQL_PORT}:3306
```
The second one is fixed, only the first can be variable. Because MySQL always will run on port 3306, inside the container.